apartment_application_service/settings.py

```diff
@@ -183,6 +186,35 @@
 CORS_ORIGIN_WHITELIST = env.list("CORS_ORIGIN_WHITELIST")
 CORS_ORIGIN_ALLOW_ALL = env.bool("CORS_ORIGIN_ALLOW_ALL")
 
+AUDIT_LOG_FILENAME = env("AUDIT_LOG_FILENAME")
+
+if AUDIT_LOG_FILENAME:
+    if "X" in AUDIT_LOG_FILENAME:
+        import random
+        import re
+        import string
+
+        system_random = random.SystemRandom()
+        char_pool = string.ascii_lowercase + string.digits
+        AUDIT_LOG_FILENAME = re.sub(
+            "X", lambda x: system_random.choice(char_pool), AUDIT_LOG_FILENAME
+        )
```

Comment:
why X is wanted to be replaced with random chars and not with date

Reply:
For consistency with Open City Profile: https://github.com/City-of-Helsinki/open-city-profile/commit/67dace7b9f17b9edce5e2ecb5b5f5897c7d6afa4